psga/s_aureus/psga.nf

```diff
@@ -0,0 +1,29 @@
+/*
+ * Main workflow for the pathogen: Staphylococcus aureus.
+ */
+params.str = 'Hello world!'
+
+process splitLetters {
+  output:
+    path 'chunk_*'
+
+  """
+  printf '${params.str}' | split -b 6 - chunk_
+  """
+}
+
+process convertToUpper {
+  input:
+    path x
+  output:
+    stdout
+
+  """
+  cat $x | tr '[a-z]' '[A-Z]'
+  """
+}
+
+
+workflow psga {
+  splitLetters | flatten | convertToUpper | view { it.trim() }
```

Comment:
Can we replace this with the code for getting the metadata samples into channels at least (see synthetic pipeline psga.nf).

Reply:
That's the next JIRA that I'm working on now